src/components/app/routes/queries/subsidies/subscriptions.js

```diff
@@ -12,32 +12,21 @@ async function fetchSubscriptionLicensesForUser(enterpriseUUID) {
   return camelCaseObject(response.data);
 }
 
-async function fetchCustomerAgreementData(enterpriseUUID) {
-  const queryParams = new URLSearchParams({
-    enterprise_customer_uuid: enterpriseUUID,
-  });
-  const config = getConfig();
-  const url = `${config.LICENSE_MANAGER_URL}/api/v1/customer-agreement/?${queryParams.toString()}`;
-  const response = await getAuthenticatedHttpClient().get(url);
-  return camelCaseObject(response.data);
-}
-
 /**
  * TODO
- * @param {*} param0
  * @returns
+ * @param enterpriseUuid
  */
 async function fetchSubscriptions(enterpriseUuid) {
-  const results = await Promise.all([
-    fetchCustomerAgreementData(enterpriseUuid),
-    fetchSubscriptionLicensesForUser(enterpriseUuid),
-  ]);
+  const response = await fetchSubscriptionLicensesForUser(enterpriseUuid);
+  // Extracts customer agreement and removes it from the original response object
+  const { customerAgreement } = response;
+  delete response.customerAgreement;
   return {
-    customerAgreement: results[0],
-    subscriptionLicenses: results[1],
+    subscriptionLicenses: response,
```

Comment:
How would you feel if we only returned the list of `results` for the `subscriptionLicenses` vs. the entire paginated response body?

The only thing in the overall response body thats nice-to-have is having the `count` already computed; however, we can also get at this from `subscriptionLicenses.length`.

That way, we don't need to do any local modifications of `response` to delete `customerAgreement` (generally, I try to avoid using `delete` if possible as to not remove properties from existing objects, which _can_ cause confusion in some cases).